src/common/include/com/amazonaws/kinesis/video/common/CommonDefs.h

```diff
@@ -686,18 +686,22 @@ extern getTName globalGetThreadName;
 //
 typedef UINT64 (*getTime)();
 
+typedef struct tm* (*getTmTime)(const time_t*);
+
 //
 // Default time library functions
 //
 #define TIME_DIFF_UNIX_WINDOWS_TIME 116444736000000000ULL
 
 PUBLIC_API UINT64 defaultGetTime();
+PUBLIC_API struct tm* defaultGetTmTime(const time_t*);

```

Comment:
Since it is public definition, I think we should name it `defaultGetThreadSafeTime` or something similar to be explicit. Also, can we add an API explanation?

Reply:
Good point, I agree.  How about `defaultGetThreadSafeTmTime` I want the tm in the name just so it's different from GetTime -- it doesn't return `UINT64` time instead it returns the `tm` struct.

Reply:
Sure! That works!